src/locale/languages/en.js

```diff
@@ -177,6 +177,8 @@ export const en = {
   },
   repository: {
     main: {
+      notFoundRepo: 'Repository is not found',
+      unknowLanguage: 'Unknown',
```

Comment:
`unknownLanguage` as @arthurdenner suggested? Sorry I am not a native English speaker.